changes/342.apichange.rst

```diff
@@ -0,0 +1 @@
+Added ``driz_args`` argument to ``resample.Resample.update_time()`` method.
```

Comment:
Would you update this to reflect the changes in this PR?

Reply:
done